alerter.cpp

```diff
@@ -24,8 +29,9 @@ void alertInCelcius(float farenheit) {
 }
 
 int main() {
-    alertInCelcius(400.5);
-    alertInCelcius(303.6);
+    alertInCelcius(400.5, networkAlertStub);
+    alertInCelcius(303.6, networkAlertStub);
+    assert(alertFailureCount == 1);

```

Comment:
This test will pass when any of the above lines increment the `alertFailureCount`. Strengthen the test to assert exactly what you want.